src/components/Hero.tsx

```diff
@@ -14,9 +15,12 @@ const Hero: React.FC<HeroProps> = ({ title, subtitle, image }) => {
         backgroundRepeat: "no-repeat"
     }
     return (
-        <div className="flex flex-col items-center justify-center h-2/3 bg-slate-600" style={bgImageStyle}>
-            <h1 className="text-5xl font-bold text-navy-blue drop-shadow-xl">{title}</h1>
-            <h2 className="text-3xl text-gray-300 drop-shadow-sm">{subtitle}</h2>
+        <div className="flex flex-col items-center justify-center h-3/4 bg-slate-600" style={bgImageStyle}>
+            <div className="space-y-4">
+                <h1 className="text-5xl font-bold text-navy-blue drop-shadow-xl text-center">{title}</h1>
```

Comment:
text color is hard to read on this background;

maybe we should abstract the text color as a prop so it's reusuable and return it to white for the landingpage hero